yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -566,7 +566,8 @@ def pixelize_cylinder(np.float64_t[:,:] buff,
     cdef np.float64_t r_inc, theta_inc
     cdef np.float64_t costheta, sintheta
     cdef int i, i1, pi, pj
-    cdef np.float64_t twoPI = 2 * np.pi
+    cdef np.float64_t onePI = np.pi
```

Comment:
here's the direct way to "import" the constant from numpy as the C level
```
cdef extern from "numpy/npy_math.h":
    double NPY_PI
```
(taken from https://github.com/yt-project/yt/pull/4948/files#r1692841029)

Reply:
(NPY_2_PI is also available)

Reply:
hah, will import `NPY_2_PI` as well in a minute after tests run this round.... i think this one will pass...

Reply:
oh wait, NPY_2_PI is 2/pi, not 2*pi ?

Reply:
aaaah, missed that, sorry